docs/core/plugins/notify_status.md

```diff
@@ -29,13 +29,23 @@ This plugin updates the `priv/status/<mailbox name>` key.
 ```[dovecot.conf]
 mail_plugins = $mail_plugins notify notify_status
 
-plugin {
-  notify_status_dict = proxy:dict-async:notify_status
+notify_status {
+  dict_driver = proxy
+  dict_proxy_name = notify_status
+  dict_proxy_socket_path = dict-async
```

Comment:
It will probably be changed in a later commit (in another MR). And I was trying to not introduce changes that I knew were coming to not run into troubles later, when porting the commits.

Thus I would disagree with changing it here, now. But I can assure you it will be updated in the near future.